<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>579</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QListWidget" name="archivoslistWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>191</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="amigoslistWidget">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>60</y>
     <width>191</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>280</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="enviarButton">
      <property name="text">
       <string>Enviar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="volverButton">
      <property name="text">
       <string>Volver</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Archivos</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>30</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Amigos</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
